### PR TITLE
variant(): add STEVAL-MKBOXPRO (SensorTile.box Pro)

### PR DESCRIPTION
**Pull request by fpistm**
This PR add the support of the new [SensorTile.box PRO](https://www.st.com/en/evaluation-tools/steval-mkboxpro.html)

### CODE REVIEW

File: variants/STM32U5xx/U575A(G-I)IxQ_U585AIIxQ/variant_STEVAL_MKBOXPRO.h [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+// #if !defined(HAL_DAC_MODULE_DISABLED)
+//   #define HAL_DAC_MODULE_ENABLED
+// #endif
+// #if !defined(HAL_OSPI_MODULE_DISABLED)
+//   #define HAL_OSPI_MODULE_ENABLED
+// #endif
+#if !defined(HAL_SD_MODULE_DISABLED)
+  #define HAL_SD_MODULE_ENABLED
+#endif
+
```
Comment (cparata) [inline variants/STM32U5xx/U575A(G-I)IxQ_U585AIIxQ/variant_STEVAL_MKBOXPRO.h:309]: Frederic,
the MDF module should be enabled here? We need it for the audio part.
BR
Carlo

Comment (fpistm) [reply variants/STM32U5xx/U575A(G-I)IxQ_U585AIIxQ/variant_STEVAL_MKBOXPRO.h:309]: Hi Carlo. Was not aware of that. I will add it.

Comment (fpistm) [reply variants/STM32U5xx/U575A(G-I)IxQ_U585AIIxQ/variant_STEVAL_MKBOXPRO.h:309]: done

**Review by cparata — APPROVED** [review]
Hello Fred,
I tested the new variant and it works with the libraries that we already have in the repo.
So, I approve it.
BR
Carlo

**Pull request merged**